todo/src/actions/index.js

```diff
@@ -0,0 +1,23 @@
+export const ADDTODO = 'ADDTODO';
+export const TOGGLETODO = 'TOGGLETODO';
+export const CLEAR = 'CLEAR'
```

Comment:
this is convention more than anything code-breaking, but stick to SCREAMING_SNAKE_CASE for the string and variable names here, and be specific as possible -- what if your app functionality expanded and you had multiple "clear" type actions you wanted to dispatch? I'd suggest either CLEAR_COMPLETED_TODO or DELETE_TODO to state the explicit action and what is being acted upon in the name